docs/docs/dev-guide-usb-logging.md

```diff
@@ -50,6 +50,13 @@ CONFIG_UART_CONSOLE_ON_DEV_NAME="CDC_ACM_0"
 CONFIG_USB_UART_DTR_WAIT=n
 ```
 
+:::note
+If you are debugging your own keyboard in your [user config repository](./user-setup.md), use
+`config/boards/shields/<your_keyboard>/<your_keyboard>.conf` instead of `app/prj.conf`. In Github
+Actions, you can search the `Kconfig file` build log to verify the options above have been enabled 
+for you successfully.
+:::
+
```

Comment:
I think this would be best placed between the code snippet and the description of where to put it. What do you think?

![image](https://user-images.githubusercontent.com/9439650/97816060-1500b600-1c58-11eb-9331-b2bbdc754fac.png)